TaskHouseApi/Controllers/WorkerController.cs

```diff
@@ -13,12 +13,12 @@ namespace TaskHouseApi.Controllers
     // base address: api/customers 
     [Authorize]
     [Route("api/[controller]")]
-    public class UsersController : Controller
+    public class WorkerController : Controller
```

Comment:
Could it have been an option to make an inhertiance-hierarchy, so that the controllers relations matches that of the models? Or would that not be able to work in practice?